geometry_msgs/Point[] horizontal_walls
geometry_msgs/Point[] vertical_walls
uint8 cnt
